src/pyfmi/fmi3.pyx

```diff
@@ -855,7 +883,15 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
         return FMI3_Type(int(variable_data_type))
 
     cdef _get_variable_description(self, FMIL3.fmi3_import_variable_t* variable):
-        cdef FMIL3.fmi3_string_t desc = FMIL3.fmi3_import_get_variable_description(variable)
+        cdef FMIL3.fmi3_string_t desc = <FMIL3.fmi3_string_t>FMIL3.fmi3_import_get_variable_description(variable)
```

Comment:
Did this really work? This is what crashed for me a few months ago which resulted in the previous implementation. I mean adding `my_type` to `foo = <my_type>bar` crashed.

Reply:
This does work; even simplified it a little more. I went with this since `desc = FMIL3. ... ` gave me conversion warnings.

However, I see that we have already had this in a couple of other places e.g., `get_model_version` etc., so this is a bit more uniform now.

Reply:
OK as long as it works for variables without any description I'm good! It was annoying previously when it crashed even though it looked identical to me between FMI 2 and 3.

Reply:
The added tests specifically contains aliases without descriptions, so it appears to work.